alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -143,6 +145,66 @@ private function appendUiConfToContent($content)
 
 		return $content;
 	}
+
+	private function appendEnvConfigToContent($content)
+	{
+
+		$protocol = (isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] == 'on') ? "https" : "http";
+
+		// The default Kaltura service url:
+		$serviceUrl = requestUtils::getApiCdnHost().'/api_v3';
+		// Default Kaltura CDN url:
+		$cdnUrl = requestUtils::getCdnHost($protocol);
+		// Default Stats URL
+		$statsServiceUrl = ($protocol == "https") ? $this->buildUrl($protocol,"stats_host_https") : $this->buildUrl($protocol,"stats_host");
```

Comment:
@yossipapi can we also get these URLs(stats services) from an infra util?

Reply:
There no function implanted that does this, but it may be useful if you want to add one.